gallery/how_to/work_with_microtvm/micro_tflite.py

```diff
@@ -192,21 +197,13 @@
 #  board to generated the right firmware image.
 #
 
-import json
-import pathlib
+boards_file = pathlib.Path(tvm.micro.get_microtvm_template_projects("zephyr")) / "boards.json"
+with open(boards_file) as f:
+    boards = json.load(f)
+
+BOARD = os.getenv("TVM_MICRO_BOARD", default="nucleo_f746zg")
+TARGET = tvm.target.target.micro(boards[BOARD]["model"])
```

Comment:
@mehrdadh This is now out of  `use_physical_hw` scope, so it will overwrite the previous `TARGET` value. Is that intentional?

Reply:
nope, that was incorrect. fixed it.
Thanks!